src/test/java/com/microsoft/sqlserver/testframework/DBSchema.java

```diff
@@ -87,6 +87,83 @@
             // Other types
         }
     }
+    
+    /**
+     * 
+     * @param autoGenerateSchema
+     * @param alternateSchema
+     */
+    DBSchema(boolean autoGenerateSchema, boolean alternateSchema) {
```

Comment:
Instead of copy paste you can use shuffle / reverse.

> 
>if(alternateSchema) { 
Collections.shuffle(sqlTypes);
or 
Collections.reverse(sqlTypes);
}
>